lib/Page.js

```diff
@@ -134,7 +134,10 @@ Page.prototype.resolveDependency = function (dependency) {
       logger.info(`Converting dynamic external resource ${file} to ${resultPath}`);
       tempPath = path.join(path.dirname(this.tempPath), '.external', path.basename(file));
     }
-    markbinder.includeFile(file)
+    markbinder.includeFile(file, null, {
```

Comment:
If you decide to swap these two parameters in the other PR, I will merge them together after that.

Reply:
Swapped in the other PR and removed here.